config/templates/pages/example.xml

```diff
@@ -126,6 +126,21 @@
             </types>
         </block>
 
+        <property name="text_line" type="text_line">
+            <meta>
+                <title lang="en">Text Line</title>
+                <title lang="de">Text</title>
+                <info_text lang="en">There are no commas allowed in this field</info_text>
+                <info_text lang="de">In diesem Feld sind keine Beistriche erlaubt</info_text>
+            </meta>
+
+            <params>
+                <param name="pattern" value="^[^,]*$"/>
+                <param name="minLength" value="2"/>
+                <param name="maxLength" value="3"/>
```

Comment:
Don't we use underscores in parameters like this in other cases? :thinking: E.g. there is `present_as` for the smart content.

Reply:
I personal would go with [maxlength](https://www.w3schools.com/tags/att_input_maxlength.asp) like its written in html inputs without any case or underscore.

Reply:
Since we never just concatenated words without a separator, I would not use `maxlength`. I agree with @danrot on keeping consistency by using snake case.
And we should probably also transform the camel case parameters of the `MediaSelection` into snake case.